docs/maintaining.md

```diff
@@ -7,7 +7,7 @@ there are some recurring tasks you may have to do to keep it operating correctly
 
 We are continuously improving Neurobagel tools and services,
 so you may want to update your Neurobagel node to the latest version to benefit from new features and bug fixes.
-We always publish our tools as [Docker images on DockerHub](https://hub.docker.com/repositories/neurobagel).
+We always publish our tools as [Docker images on DockerHub](https://hub.docker.com/u/neurobagel).
```

Comment:
interesting. did that change recently?

Reply:
I guess! We were also using `https://hub.docker.com/u/neurobagel` elsewhere in the docs 🤷